### PR TITLE
LPS-192023 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://liferay.atlassian.net/browse/LPS-192023

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase [hunk 1]
```diff
@@ -886,8 +882,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LPS-105156. Authorized application should be automatically removed from the User's connect table when it is expired. Blocked by LPS-191344."
-	@ignore = "true"
+	@description = "This is a use case for LPS-105156. Authorized application should be automatically removed from the User's connect table when it is expired."
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase:885]: This test case still failed at CI run, this one should remain in quarantine while we make it more stable

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]

**Comment by melodywu** [comment]
ci:test:security

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase (hunk 1 above)
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase:885]: Ok  and done

**Comment by melodywu** [comment]
ci:test:security

**Comment by melodywu** [comment]
ci:test:security

**Review by gaborlovasdotcom — APPROVED** [review]
LGTM

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**